config/settings.yml

```diff
@@ -27,6 +27,11 @@ github_notification_monitor:
   repo_names: []
 merge_target_titler:
   enabled_repos: []
+path_based_labeler:
+  enabled_repos: []
+  rules: []
+    # - pattern:
+    #   label:
```

Comment:
We might want different rules per repo.  This is because the labels for one repo are not likely to be the same for other repos.  This is, in fact, already the case with ManageIQ/manageiq versus ManageIQ/manageiq-ui-classic, for example.